benchmarks/benchmark8.ipynb

```diff
@@ -129,20 +129,92 @@
+    "\n",
+    " - The solid fraction in the domain (the integral $\\int_V \\phi dV / \\int_V dV$)\n",
+    " \n",
+    " - The free energy, $\\mathcal{F}$\n",
+    " \n",
+    "One PFHub upload is required with two data contributions for each of the three simulations (for a total of six data contributions). The data contributions should be named as `solid_fraction_1` and `free_energy_1`, `solid_fraction_2` and `free_energy_2` as well as `solid_fraction_3` and `free_energy_3` where `_1`, `_2` and `_3` refers to $r_0 = r^*$, $r_0 = 0.99 r^*$, and $r_0 = 1.01 r^*$, respectively. The data contributions should be stored in either CSV of JSON format. The columns name for time and quantity need to be identified correctly on the upload form for each data contribution. The data should probably be stored in three separate files for each simulation and each file with three columns for time, free energy and solid fraction.\n",
+    "\n",
+    "Further data to upload can include a Youtube video, images of the nucleus at different times or the entire phase field variable at different times. These are not required, but will help others view your work.\n",
+    "\n",
+    "### Part (b)\n",
+    "\n",
+    "Follow the time evolution of the system for $t = 400$ units. The following data should be collected for upload as frequently as possible.\n",
+    "\n",
+    " - The solid fraction in the domain (the integral $\\int_V \\phi dV / \\int_V dV$)\n",
+    " \n",
+    " - The free energy, $\\mathcal{F}$\n",
+    " \n",
+    "Two data contributions are required and should be named as `solid_fraction` and `free_energy`, and stored in either CSV or JSON format. The results can be stored in two separate files or one single file as long as each column name for time and quantity is identified on the upload form. In addition, the phase field snapshot at time $t=80$ is required. The data should be named as `image` and stored in `PNG` or `TIFF` format.\n",
+    "\n",
+    "### Part (c)\n",
+    "\n",
+    "Follow the time evolution of the system for $t = 600$ units. The following data should be collected for upload as frequently as possible.\n",
+    "\n",
+    " - The solid fraction in the domain (the integral $\\int_V \\phi dV / \\int_V dV$)\n",
+    " \n",
+    " - The free energy, $\\mathcal{F}$\n",
+    " \n",
+    "Two data contributions are required and should be named as `solid_fraction` and `free_energy`, and stored in either CSV or JSON format. The results can be stored in two separate files or one single file as long as each column name for time and quantity is identified on the upload form. In addition, the phase field snapshot at time $t=100$ is required. The data should be named as `image` and stored in `PNG` or `TIFF` format.\n",
```

Comment:
Can we choose a less-generic name for this specifically-required image? Say, `phase-field-100.png` or `phase-field-100.tif`?

Taken literally, the uploads will *not* have a file extension, which is uncommon. Is this intended?

Reply:
"In addition, the phase field snapshot at time $t=100$ is required. Please upload an image in either PNG or TIFF format, named `phase-field-100.png` or `phase-field-100.tif`."

Reply:
We're not naming the data file here, but the data set. The data file can be called anything. The data set name is now required to be `phase-field-80` since we upload at $t=80$.